components/support/nimbus-fml/src/workflows.rs

```diff
@@ -180,17 +180,46 @@ mod test {
         })
     }
 
+    fn generate_multiple_and_assert(
+        test_script: &str,
+        manifests: &[&str],
+        channel: &str,
+    ) -> Result<()> {
+        let cmds = manifests.iter().map(|manifest| {
+            let cmd = create_command_from_test(test_script, manifest, channel, false)?;
+            generate_struct(&cmd)?;
+
+            Ok(cmd)
+        });
+
+        let cmds: Result<Vec<GenerateStructCmd>> = cmds.into_iter().collect();
+        let cmds = cmds?;
+        let first = cmds
+            .first()
+            .expect("At least one manifests are always used");
+        let language = first.language;
+
+        let manifests_out = cmds
+            .iter()
+            .map(|cmd| cmd.output.display().to_string())
+            .collect::<Vec<_>>();
+
+        run_script_with_generated_code(language, &manifests_out, test_script)?;
+        Ok(())
+    }
+
     fn run_script_with_generated_code(
         language: TargetLanguage,
-        manifest_out: String,
+        manifests_out: &[String],
         test_script: &str,
     ) -> Result<()> {
         match language {
             TargetLanguage::Kotlin => {
-                kotlin::test::run_script_with_generated_code(manifest_out, test_script)?
+                kotlin::test::run_script_with_generated_code(manifests_out, test_script)?
             }
             TargetLanguage::Swift => swift::test::run_script_with_generated_code(
-                manifest_out.as_ref(),
+                // TODO we don't support Swift yet!
```

Comment:
I see that we have a `test_importing_ios` test below, is this comment still true? and if so, Is this planned as a part of this patch, or will we have a ticket on file to add swift tests?